### DISCUSSION TITLE
Client-Side Routes: "The page has expired" Error on Login

### DISCUSSION THREAD

**Discussion by fredikaputra** [livewire/livewire#6416] [1. Help] [answered]
I'm currently working on a project using Livewire v3 beta and have encountered an issue related to session management and client-side routes.

**Issue:**
After logging out a user from my Livewire application, I'm facing a problem with the login process. Upon attempting to log in again, I consistently receive a "The page has expired" error. This issue specifically arises when I use client-side routes.

**Logout Process (Simplified):**
```php
public function logout()
{
    Auth::logout();
    session()->invalidate();
    session()->regenerateToken();
    return $this->redirect('/', true);
}
``` 

**Troubleshooting Steps Taken:**

- I've tried removing the `session()->invalidate();` and `session()->regenerateToken();` lines, which seems to make the client-side routes work again, but I understand that these lines are related to security measures.

- I've reviewed the Livewire documentation and forums, but I haven't found a direct solution to this issue.

Should I take out these lines? I get that CSRF protection is important, so I don't want to just remove them without knowing what might happen.

**Comment by danie-ramdhani** [comment]
try remove the navigate to false:
```php
return $this->redirect(Login::class, false);
```

**Reply by fredikaputra** [reply]
Appreciate the input! I gave it a shot with navigate set to false, and you're right, it did the trick. However, I'm on a mission to make the most of client-side routing with navigate set to true for that seamless experience. My only concern is keeping things watertight on the security side.

**Comment by joshhanley** [comment, marked as answer]
This has been fixed, see https://github.com/livewire/livewire/discussions/6784#discussioncomment-6982709
